docs/notebooks/figanos_hvplot.ipynb

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.                                          #'group', 
</pre><p>clean up the commented line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>